django_lightweight_queue/job.py

```diff
@@ -1,21 +1,38 @@
 import sys
 import json
 import time
+import datetime
 
 from .utils import get_path, get_middleware
 
 class Job(object):
-    def __init__(self, path, args, kwargs, timeout=None, sigkill_on_stop=False):
+    def __init__(
+        self,
+        path,
+        args,
+        kwargs,
+        timeout=None,
+        sigkill_on_stop=False,
+        created_time=None,
+    ):
         self.path = path
         self.args = args
         self.kwargs = kwargs
         self.timeout = timeout
         self.sigkill_on_stop = sigkill_on_stop
+        self.created_time = (
+            created_time or datetime.datetime.utcnow().isoformat(sep=' ')
```

Comment:
a8411f6 removes the argument and changes `from_json` to match, 14bca62 introduces a constant for the time format and parsing so that the JSON uses a string and the Job instances have a `datetime`.